src/Styles.js

```diff
@@ -102,7 +102,7 @@ a:hover {
 .separator {
   background-color: var(--api-summary-separator-color, rgba(0, 0, 0, 0.12));
   height: 1px;
-  margin: 40px 0;
+  margin: var(--api-summary-separator-margin-y, 40px) var(--api-summary-separator-margin-x, 0);
```

Comment:
Why not just `--api-summary-separator-margin` with a default value of `40px 0`? I don't think it's beneficial to split it into two variables.

Reply:
Sure, I can do that